snekbox/nsjail.py

```diff
@@ -188,7 +189,12 @@ def python3(
             py_args: Arguments to pass to Python.
             files: FileAttachments to write to the sandbox prior to running Python.
             nsjail_args: Overrides for the NsJail configuration.
+            binary_path: The path to the binary to execute under.
+                If None, defaults to "/lang/python/default/bin/python"
```

Comment:
I think we should rename `/lang` since it's going to be more prominent now. Maybe `/snekbin`? — doubles as a kinda clever play on "snekbox". If so, we also have to rename our references to the path in our CONTRIBUTING.md docs.

Reply:
Yeah I agree this is a good change https://github.com/python-discord/snekbox/pull/196/commits/f726695b1a1d99b5c61ea09bd3a364103a08d7bf